dbms/src/Flash/Pipeline/Schedule/Events/Event.cpp

```diff
@@ -145,13 +145,7 @@ void Event::scheduleTasks()
         unfinished_tasks);
     if (!tasks.empty())
     {
-        // If query has already been cancelled, we can skip scheduling tasks.
-        // Then tasks will be destroyed and call `onTaskFinish`.
-        if (likely(!exec_status.isCancelled()))
```

Comment:
Why we not check this status any more?

Reply:
Because all tasks must call `finalize`, they will still be submitted to `TaskScheduler` here.